src/selectors/getTip.ts

```diff
@@ -0,0 +1,7 @@
+import State from '../@types/State'
+import TipId from '../@types/TipId'
+
+/** Gets the tip to be displayed. */
+export const getTip = (state: State): TipId | undefined => state.tips[0]
```

Comment:
Let's inline this function in `Tips.tsx` since it's not used anywhere else.

Reply:
Inlined: 4f77d17701c72c441f33747f879fbf2c3136942e